osu.Game/Configuration/SettingsStore.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using osu.Game.Database;
+
+namespace osu.Game.Configuration
+{
+    public class SettingsStore : DatabaseBackedStore
+    {
+        public event Action SettingChanged;

```

Comment:
This doesn't look to be used.

Reply:
Again, was just copying the way `KeyBindingStore` is laid out. It will likely be used in the future to trigger updates of the settings screen, just like it is in `KeyBindingStore`